### PR TITLE
[168275]- Grievance: Enable grievance approver to mark individuals as unique when resolving NA

### PR DESCRIPTION
**Pull request by pavlo-mk**
[Enhancement 168275](https://unicef.visualstudio.com/ICTD-HCT-MIS/_workitems/edit/168275): Grievance - Enable grievance approver to mark individuals as unique when resolving a needs adjudication grievance

### CODE REVIEW

**Comment by mmaciekk** [comment]
@domdinicola 
Here is the pull request regarding marking individuals as unique/distinct

**Review by patryk-dabrowski — APPROVED** [review]

File: compose.selenium.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-version: '3.7'
+version: "3.7"
```
Comment (wozniakpl) [inline compose.selenium.yml:1]: I think version is deprecated and can be safely removed.

File: backend/hct_mis_api/apps/grievance/services/needs_adjudication_ticket_services.py [hunk 2]
```diff
@@ -216,34 +231,37 @@ def mark_as_duplicate_individual_and_reassign_roles(
...
+                ticket_details.role_reassign_data,
+                user,
+                ticket_details.ticket.programs.all(),
+                "new_individual",
+            )
     else:
-        household = reassign_roles_on_disable_individual_service(
-            individual_to_remove, ticket_details.role_reassign_data, user, ticket_details.ticket.programs.all()
-        )
+        if need_reassign_roles:
```
Comment (wozniakpl) [inline backend/hct_mis_api/apps/grievance/services/needs_adjudication_ticket_services.py:248]: elif

File: backend/selenium_tests/grievance/grievance_tickets/test_grievance_tickets.py [hunk 3]
```diff
@@ -204,3 +348,112 @@ def test_grievance_tickets_create_new_ticket_referral(
...
+        pageIndividuals: Individuals,
+    ) -> None:
+        pageGrievanceTickets.getNavGrievance().click()
+        assert "Grievance Tickets" in pageGrievanceTickets.getGrievanceTitle().text
+        pageGrievanceTickets.getTabSystemGenerated().click()
+        pageGrievanceTickets.getTicketListRow()[0].click()
+        pageGrievanceDetailsPage.getSelectAllCheckbox().click()
+        pageGrievanceDetailsPage.getPersonIcon()
+        assert "person-icon" in [
+            ii.get_attribute("data-cy")
```
Comment (wozniakpl) [inline backend/selenium_tests/grievance/grievance_tickets/test_grievance_tickets.py:366]: consider something more verbose than `ii`

**Review by wozniakpl — APPROVED** [review]

File: backend/selenium_tests/grievance/grievance_tickets/test_grievance_tickets.py (hunk 3 above)
Comment (pavlo-mk) [reply backend/selenium_tests/grievance/grievance_tickets/test_grievance_tickets.py:366]: I will update it in separate PR later today or tomorrow morning

File: backend/hct_mis_api/apps/grievance/services/needs_adjudication_ticket_services.py (hunk 2 above)
Comment (pavlo-mk) [reply backend/hct_mis_api/apps/grievance/services/needs_adjudication_ticket_services.py:248]: I will update it in separate PR later today or tomorrow morning

File: compose.selenium.yml (hunk 1 above)
Comment (pavlo-mk) [reply compose.selenium.yml:1]: I will update it in separate PR later today or tomorrow morning

**Pull request merged**